study.md

```diff
@@ -21,18 +21,18 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+I wouldn't say I'm confused by callback functions, I just need some more time to understand their purpose and their usefulness.  I need a little more practice with understanding the nuances of scope in JS.
```

Comment:
We'll be going over array methods this coming week and many of them have callback functions as parameters.